ethereum/core/src/main/java/tech/pegasys/teku/core/epoch/EpochProcessor.java

```diff
@@ -27,7 +27,7 @@
    */
   public static BeaconState processEpoch(final BeaconState preState)
       throws EpochProcessingException {
-    final ValidatorStatuses validatorStatuses = ValidatorStatuses.create(preState);
+    final ValidatorStatuses validatorStatuses = ValidatorStatuses.createTotalBalances(preState);
```

Comment:
nit: I don't think you meant to rename this method - IntelliJ was being "helpful".

Reply:
oh shoot yeah. reverted.